jmx scraper: add missing relevant config options (#1979)
open-telemetry/opentelemetry-java-contrib: jmx-scraper/README.md

| config option                 | default value   | description                                                                                                                                                       |
|-------------------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `otel.metric.export.interval` | `1m` (1 minute) | metric export interval, also controls the JMX sampling interval                                                                                                   |
| `otel.metrics.exporter`       | `otlp`          | comma-separated list of metrics exporters supported values are `otlp` and `logging`, additional values might be provided through extra libraries in the classpath |
| `otel.service.name`           |                 | service name                                                                                                                                                      |

> | `otel.resource.attributes`    |                 | resource attributes, allow to to provide [service attributes](https://opentelemetry.io/docs/specs/semconv/registry/attributes/service/)                           |

Reviewer: nit: I don't know if I'd limit the description here to just service attributes. 

```suggestion
| `otel.resource.attributes`    |                 | used to specify otel resource attributes, including service attributes. See [the sdk configuration[(https://opentelemetry.io/docs/languages/java/configuration/#properties-resource) and [service attributes](https://opentelemetry.io/docs/specs/semconv/registry/attributes/service/)                           |
```)

| config option                 | default value   | description                                                                                                                                                       |
|-------------------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `otel.metric.export.interval` | `1m` (1 minute) | metric export interval, also controls the JMX sampling interval                                                                                                   |
| `otel.metrics.exporter`       | `otlp`          | comma-separated list of metrics exporters supported values are `otlp` and `logging`, additional values might be provided through extra libraries in the classpath |
| `otel.service.name`           |                 | service name                                                                                                                                                      |

> | `otel.resource.attributes`    |                 | used to specify otel resource attributes, including service attributes. See [the sdk configuration[(https://opentelemetry.io/docs/languages/java/configuration/#properties-resource) and [service attributes](https://opentelemetry.io/docs/specs/semconv/registry/attributes/service/)                           |

Reviewer: Whoops!
```suggestion
| `otel.resource.attributes`    |                 | used to specify otel resource attributes, including service attributes. See [the sdk configuration](https://opentelemetry.io/docs/languages/java/configuration/#properties-resource) and [service attributes](https://opentelemetry.io/docs/specs/semconv/registry/attributes/service/)                           |
```

> ```)

Reviewer: Whoops!
```suggestion
```
